### PR TITLE
Add function Save-FinOpsHubTemplate

### PR DESCRIPTION
**Pull request by aromano2**
## 🛠️ Description

Fixes #(issue)

## 🔬 How has this been tested?

- [ ] 🫰 PS -WhatIf / az validate
- [ ] 👍 Manually deployed + verified
- [ ] 💪 Unit tests

### CODE REVIEW

File: src/powershell/FinOpsToolkit.psm1 [hunk 1]
```diff
@@ -1,28 +1,144 @@
...
+        [string]
+        $Destination = $env:temp,
+
+        [Parameter()]
+        [switch]
+        $Force
+    )
+
+    New-Directory -Path $Destination
+    $releases = Get-FinOpsToolkitVersion -Latest:$($Version -eq 'Latest') -Preview:$Preview
```
Comment (flanakin) [inline src/powershell/FinOpsToolkit.psm1:78]: Is `$Version -eq 'Latest'` case-sensitive? The value you set it to is lowercase above.

File: src/powershell/FinOpsToolkit.psm1 [hunk 2]
```diff
@@ -1,28 +1,144 @@
...
+
+    if ($Version -eq 'Latest')
```
Comment (flanakin) [inline src/powershell/FinOpsToolkit.psm1:80]: Shouldn't you factor `$Preview` into this logic?

**Review by flanakin — APPROVED** [review]

File: src/powershell/FinOpsToolkit.psm1 [hunk 3]
```diff
@@ -1,28 +1,144 @@
...
+        Save-FinOpsHubTemplate
+
+        Downloads the latest version of FinOps hub template to current users' temp folder.
+
+    .EXAMPLE
+        Save-FinOpsHubTemplate -Version '1.0.0' -Destination 'C:\myHub' -Force
+
+        Downloads version 1.0.0 of FinOpsHub template to c:\myHub directory. It will overwrite an existing 1.0.0.zip file if it exists.
+#>
+function Save-FinOpsHubTemplate
```
Comment (flanakin) [inline src/powershell/FinOpsToolkit.psm1:55]: Should we add tests for this? I know we don't have the infrastructure for that yet, but even if it's a placeholder, that might still be nice until we get the infrastructure in place. Thoughts?

File: src/powershell/FinOpsToolkit.psm1 (hunk 2 above)
Comment (aromano2) [reply src/powershell/FinOpsToolkit.psm1:80]: it is, I pass it in to Get-FinOpsToolkitVersion

**Pull request merged**